src/main/kotlin/org/rust/ide/annotator/fixes/ChangeReturnTypeFix.kt

```diff
@@ -27,6 +28,7 @@ import org.rust.lang.core.types.ty.TyUnknown
 
 class ChangeReturnTypeFix(
     element: RsElement,
+    @SafeFieldForPreview
```

Comment:
Do I understand correctly that `Ty` type is safe even if it may reference a physical PSI just because it's a read only property?

Reply:
Yes, because we don't modify that physical PSI and use it to generate some `String`